app/src/main/java/agency/five/codebase/android/movieapp/ui/main/MainScreen.kt

```diff
@@ -0,0 +1,217 @@
+                        id = R.string.app_name
+                    )
+                )
+            }
+        )
+    } else {
+        CenterAlignedTopAppBar(
+            colors = TopAppBarDefaults.centerAlignedTopAppBarColors(containerColor = Blue),
+            title = {
+                Image(
+                    painter = painterResource(id = R.drawable.tmdb_logo),
+                    contentDescription = stringResource(
+                        id = R.string.app_name
+                    )
+                )
+            }
+        )
+    }
+}
+
+@Composable
+private fun BackIcon(
+    onBackClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    IconButton(
+        onClick = { onBackClick() },
+        modifier = modifier
+    ) {
+        Icon(
+            painter = painterResource(id = R.drawable.ic_baseline_arrow_back_ios_24),
+            contentDescription = stringResource(id = R.string.back_icon),
+            tint = Color.White
+        )
+    }
+}
+
+@Composable
+private fun BottomNavigationBar(
+    destinations: List<NavigationItem>,
+    onNavigateToDestination: (NavigationItem) -> Unit,
+    currentDestination: NavDestination?,
+) {
+    BottomNavigation(
+        backgroundColor = MaterialTheme.colors.background
+    ) {
+        destinations.forEach { destination ->
+            val selected = destination.route == currentDestination?.route
+            BottomNavigationItem(
+                selected = selected,
+                selectedContentColor = Blue,
+                unselectedContentColor = Color.Gray,
+                icon = {
+                    Column(horizontalAlignment = CenterHorizontally) {
+                        if (selected) {
+                            Icon(
+                                painter = painterResource(id = destination.selectedIconId),
+                                contentDescription = stringResource(id = R.string.home)
+                            )
+                            Text(
+                                text = stringResource(id = destination.labelId),
+                                maxLines = 1,
+                                overflow = TextOverflow.Ellipsis,
+                                style = BottomNavText
+                            )
+                        } else {
+                            Icon(
+                                painter = painterResource(id = destination.unselectedIconId),
+                                contentDescription = stringResource(id = R.string.favorites)
+                            )
+                            Text(
+                                text = stringResource(id = destination.labelId),
+                                maxLines = 1,
+                                overflow = TextOverflow.Ellipsis,
+                                style = BottomNavText
+                            )
+                        }
+                    }
+                },
```

Comment:
This isn't necessary `BottomNavigationItem` has parameter label which should be used to set text of item. 
You could also use `if else`  to just choose resource instead, because you are using the same composable.
`Icon(painter = painterResource(id = if (selected) selectedId else unselectedId)`,